src/database/schemas.ts

```diff
@@ -35,10 +33,9 @@ export default class Schemas {
       switch (type) {
         case `schemas`:
           selects.push([
-            `select '${type}' as OBJ_TYPE, SCHEMA_NAME as NAME, SCHEMA_TEXT as TEXT, SYSTEM_SCHEMA_NAME as SYS_NAME, '' as SYS_SCHEMA, '' as SPECNAME, '' as BASE_SCHEMA, '' as BASE_OBJ`,
+            `select '${type}' as OBJ_TYPE, QSYS2.DELIMIT_NAME(SCHEMA_NAME) as NAME, SCHEMA_TEXT as TEXT, SYSTEM_SCHEMA_NAME as SYS_NAME, '' as SYS_SCHEMA, '' as SPECNAME, '' as BASE_SCHEMA, '' as BASE_OBJ`,
```

Comment:
Delimiting the schema name here breaks the SchemaBrowser `vscode-db2i.manageSchemaBrowserList` action, which itself applies delimiting to the schema names.  That then trickles through the schema browser tree as the name ends up double-delimited.  The SchemaBrowser handles the sorting of the schemas in its `getChildren` method.